_commands/basics/kill.md

```diff
@@ -17,8 +18,32 @@ TODO: Add documentation for this command by submitting a pull request.
 
 #### Example command
 
-##### Break it down
+##### kill -9 PID
+Actually, kill is a command to send signals to the specified process, each process has a set of standard handlers by the operating systems, and signal 9 called SIGKILL can almost guarantee to kill a process if the default signal 15 can't kill it.
```

Comment:
With any instance of the command, it should be styled as `inline code` using backticks (`). So the beginning of your documentation should look something like

>Actually, `kill` is a command...